pulp_container/tests/functional/conftest.py

```diff
@@ -0,0 +1,79 @@
+from urllib.parse import urljoin
+
+import pytest
+import requests
+
+from pulp_container.tests.functional.utils import (
+    TOKEN_AUTH_DISABLED,
+    AuthenticationHeaderQueries,
+    BearerTokenAuth,
+)
+
+
+@pytest.fixture
+def get_response(bindings_cfg):
+    """Return a response considering token authentication."""
```

Comment:
I kind of expect mount blob to use this fixture. But it's using requests.post instead of get. Can we implement these in a way that we can call `local_registry.get(relative_url)` and `local_registry.post(relative_path, payload)`?

Reply:
Yes, I can work on that.